### PR TITLE
Removing local cluster from graphcool.yml

### PR DESCRIPTION
**Pull request by qucode1**
- Remove local cluster option from 1.0 examples, to allow choosing a cluster on deployment.

- update download link in permissions example

### CODE REVIEW

File: examples/1.0/file-api/database/graphcool.yml [hunk 1]
```diff
@@ -6,4 +6,3 @@ cluster: local
 datamodel:
   - datamodel.graphql
   
-secret: ${env:GRAPHCOOL_SECRET}
```
Comment (marktani) [inline examples/1.0/file-api/database/graphcool.yml]: why did you remove the secret here?

Comment (qucode1) [reply examples/1.0/file-api/database/graphcool.yml]: sorry that is not what I had in mind. I will fix that.

**Comment by marktani** [comment]
Thanks a lot @qucode1 🎉
